src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1098,7 +1109,8 @@ private void EnforceDoesNotReturn(SyntaxNode? syntaxOpt)
             bool useConstructorExitWarnings,
             VariableState? initialNullableState,
             bool getFinalNullableState,
-            out VariableState? finalNullableState)
+            out VariableState? finalNullableState,
+            bool requiredAnalysis = true)
```

Comment:
Nit: this threw me off a bit because i kept reading this variable name in the past tense and couldn't make sense of it. Once I got to the bottom of the file i realized it meant more "analysis is required". 

Not sure if this threw others off or not.

Reply:
Renamed to `requiresAnalysis`.

---
In reply to: [551607093](https://github.com/dotnet/roslyn/pull/49953#discussion_r551607093) [](ancestors = 551607093)